packages/optimistic-oracle/src/proposer.js

```diff
@@ -220,19 +236,35 @@ class OptimisticOracleProposer {
         transactionConfig: { ...this.gasEstimator.getCurrentFastPrice(), from: this.account },
       });
 
-      const logResult = {
-        tx: receipt.transactionHash,
-        requester: receipt.events.ProposePrice.returnValues.requester,
-        proposer: receipt.events.ProposePrice.returnValues.proposer,
-        identifier: this.hexToUtf8(receipt.events.ProposePrice.returnValues.identifier),
-        ancillaryData: receipt.events.ProposePrice.returnValues.ancillaryData || "0x",
-        timestamp: receipt.events.ProposePrice.returnValues.timestamp,
-        proposedPrice: receipt.events.ProposePrice.returnValues.proposedPrice,
-        expirationTimestamp: receipt.events.ProposePrice.returnValues.expirationTimestamp,
-      };
+      let logResult;
+      if (this.optimisticOracleClient.oracleType === "SkinnyOptimisticOracle") {
+        logResult = {
+          tx: receipt.transactionHash,
+          requester: receipt.events.ProposePrice.returnValues.requester,
+          proposer: receipt.events.ProposePrice.returnValues.request.proposer,
+          identifier: this.hexToUtf8(receipt.events.ProposePrice.returnValues.identifier),
+          ancillaryData: receipt.events.ProposePrice.returnValues.ancillaryData || "0x",
+          timestamp: receipt.events.ProposePrice.returnValues.timestamp,
+          request: receipt.events.ProposePrice.returnValues.request,
+          proposedPrice: receipt.events.ProposePrice.returnValues.request.proposedPrice,
```

Comment:
oof to this change. otherwise, they are identical

Reply:
yep...